build/AzureInfo.props

```diff
@@ -1,6 +1,10 @@
 <Project>
   <PropertyGroup>
     <Product>Sdk</Product>
+    <CoreSetupBlobRootUrl Condition="'$(PB_AssetRootUrl)' != ''">$(PB_AssetRootUrl)</CoreSetupBlobRootUrl>
```

Comment:
Could would we not simply set /p:CoreSetupBlobRootUrl=$(PB_AssetRootUrl) on the msbuild line and remove this code here instead?

Reply:
MSRC builds pass in 'CoreSetupBlobRootUrl'. See: https://devdiv.visualstudio.com/DevDiv/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=7080

That would result in setting the property to an empty string in the MSRC case.